lib/ricohapi/mstorage/client.rb

```diff
@@ -90,7 +92,17 @@ def delete(media_id)
 
       # PUT /media/{id}/meta/user/{key}
       def add_meta(media_id, user_meta)
-        # TODO: do something
+        raise Error.new("user_meta is empty: nothing to request.") if user_meta.empty?
```

Comment:
I'm going to use `"Invalid parameter: #{media_id.inspect}, #{user_meta.inspect}: nothing to request."`to show parameter contents.
Also, I'm going to use `unless user_meta` because it makes sense that this method does not need to raise any errors when the `user_meta` is empty.